tests/integration/cgroups.bats

```diff
@@ -95,7 +95,7 @@ EOF
 
     runc run -d --console-socket $CONSOLE_SOCKET test_cgroups_permissions
     [ "$status" -eq 1 ]
-    [[ ${lines[1]} == *"cannot set pids limit: container could not join or create cgroup"* ]]
+    [[ ${lines[1]} == *"permission denied"* ]] || [[ ${lines[1]} == *"cannot set pids limit: container could not join or create cgroup"* ]]
```

Comment:
Can you tell what exact "permission denied" error are we getting here? If it's not descriptive enough, maybe makes sense to wrap it (merely "permission denied" looks way too generic).


We can wrap the generic "permission denied" error to avoid

Reply:
updated